### ISSUE TITLE
Support systemd-firstboot

### ISSUE THREAD

**Issue by shibumi** [systemd/mkosi#384]
Hi,
I try to generate raw_gpt images with enabled systemd-firstboot.

Here are my files:

minimal.default
```
[Distribution]
Distribution=arch

[Output]
Format=raw_gpt
Bootable=yes
Output=minimal.img
```

minimal.postinst:
```
#!/bin/bash

cat > /etc/systemd/system/systemd-firstboot.service << EOF
[Unit]
Description=First Boot Wizard
Documentation=man:systemd-firstboot(1)
DefaultDependencies=no
Conflicts=shutdown.target
After=systemd-remount-fs.service
Before=systemd-sysusers.service sysinit.target shutdown.target
ConditionPathIsReadWrite=/etc
ConditionFirstBoot=yes

[Service]
Type=oneshot
RemainAfterExit=yes
ExecStart=/usr/bin/systemd-firstboot --prompt --setup-machine-id
StandardOutput=tty
StandardInput=tty
StandardError=tty

[Install]
WantedBy=sysinit.target
EOF

systemctl enable systemd-firstboot.service
rm /etc/{machine-id,localtime,shadow,locale.conf}
```

I build them via: 
```
sudo mkosi --default minimal/minimal.default --postinst minimal/minimal.postinst --root-size=2G --force
```

The build works so far, what is not working is booting into `systemd-firstboot`.
When the VM boots up, the system will have machine-id file and therefore it's not possible to boot into systemd-firstboot. Here is my qemu line: 

```
qemu-system-x86_64 -m 512 -smp 2 -bios /usr/share/ovmf/x64/OVMF_CODE.fd -drive format=raw,file=minimal.img
```

**Comment by DaanDeMeyer** [comment]
We explicitly remove the machine-id from the final image now so this should work, let me know if it doesn't and I can reopen

**Comment by shibumi** [comment]
Hi @DaanDeMeyer  I tried this and it doesn't work. The VM is not booting into firstboot mode.

**Comment by behrmann** [comment]
@shibumi Fixed by #1118
